client/src/redux/store.js

```diff
@@ -12,7 +13,14 @@ const hydrateStore = () => {
 let middleware = [thunkMiddleware, promiseMiddleware()];
 
 if (process.env.NODE_ENV !== 'production') {
-  middleware.push(createLogger());
+  if (process.env.REACT_APP_DEBUG_MIDDLEWARE === 'true') {
+    middleware.push(createLogger());
+  }
+  cookies.set(
+    'csrftoken',
+    'xLPbEvorrDHqC9poUjUpKtJcf2kmskkwVzHpkiwqO57zD9ViBtrtqEud6llTnVPH'
+  );
+  console.warn('Warning: Loading spoof auth token.');
```

Comment:
Note: In a follow up PR we could probably investigate some development "helper" functions that handle these one-offs dev checks.